README.md

```diff
@@ -1,2 +1,174 @@
-# recipe-app
-This is a Ruby on Rails web app where a user can post their recipe and foods, and can decide to share it in public or make it private
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Recipe App</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [Diagram](#diagram)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Recipe App <a name="about-project"></a>
+
+
+**Recipe App** keeps track of all your recipes and ingredients. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+### ERD Diagram <a name="diagram">
+![Preview](./app/assets/images/recipe-app.png)
+
+### Demo
+
+Click [here](https://recipe-ruby.onrender.com) to view our app
+
```

Comment:
- [ ] @blase147 and @kaizipaul great work implementing the changes requested however since this is not a project requirement I would advice you delete the whole section, or fix the error on render shown below
![image](https://user-images.githubusercontent.com/61408860/222753585-7b71b57b-52f6-49d5-a4da-272e9164bed0.png)